packages/ts/src/components/area/index.ts

```diff
@@ -44,6 +47,30 @@ export class Area<Datum> extends XYComponentCore<Datum, AreaConfigInterface<Datu
     this.stacked = Array.isArray(this.config.y)
   }
 
+  get bleed (): Spacing {
```

Comment:
This is taken from https://github.com/f5/unovis/blob/9ef4221750f04783708ffa3d41525c2d0f4038af/packages/ts/src/components/line/index.ts#L47-L67